setup.py

```diff
@@ -26,7 +26,7 @@
 
 DEPENDENCIES = [
     'msrestazure',
-    'azure-mgmt-hanaonazure==0.1.1'
+    'azure-mgmt-hanaonazure==0.2.0'
```

Comment:
Why is this changing to "2.0", but line 11 is changing from "1.1" to "1.2"?

Reply:
0.1.2 is the new extension version
2.0 is the new SDK version

Reply:
Oh, weird. So the SDK version is out of our hands, yes? Should we just change the "1.2" version to "2.0" to match? Is that under our control?

Reply:
The SDK version is out of our control
https://pypi.org/project/azure-mgmt-hanaonazure/0.2.0/

We could change our version to whatever we want, I can change it to 2.0 to match the SDK version if you want

Reply:
Unless you have any objections, I think this is the best option for us. We have to bump the version to something, and as someone who has correlated version numbers between 3 different packages in a previous project, it's not worth it.

Reply:
wait what? no!
We don't own the SDK, it is a dependency, we could very well have bumped the extension version without changing the SDK one.
Now, if we want to change the extension tomorrow (and bump version) and the SDK haven't released this week, are we forced to keep 2.0 to keep on par with the SDK? should we bump to 2.1?  I think we should decouple those versions.
*strongly opposed*

Reply:
Jorge has a point here; I would be fine if you kept it as a bump to 0.1.2.